docs/machine-learning/resources/tasks.md

```diff
@@ -7,7 +7,7 @@ author: natke
 ---
 # Machine learning tasks in ML.NET
 
-When building a machine learning model, you first need to define what you are hoping to achieve with your data. This allows you to choose the right machine learning task for your situation. The following list describes the different machine learning tasks that you can choose from and some common use cases.
+When building a machine learning model, you first need to define what you are hoping to achieve with your data. This allows you to choose the right machine learning task for your situation. The following list describes the different machine learning tasks that you can choose from and some common use cases.  See [Algorithms](how-to-choose-an-ml-net-algorithm.md) for more information about choosing the task that is appropriate for your scenario.
```

Comment:
```suggestion
When building a machine learning model, you first need to define what you are hoping to achieve with your data. This allows you to choose the right machine learning task for your situation. The following list describes the different machine learning tasks that you can choose from and some common use cases.  See [Algorithms](../how-to-choose-an-ml-net-algorithm.md) for more information about choosing the task that is appropriate for your scenario.
```

Reply:
@stevenjpr : Hi, Steven, More good edits. The link didn't work, but if you can accept my change we're good to go. Thank you!